frontend/app-development/features/latestCommit/LatestCommit.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect } from 'react';
+import { StudioPageSpinner } from '@studio/components';
+import { gitCommitPath } from 'app-shared/api/paths';
+import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmentParams';
+import { useBranchStatusQuery } from 'app-development/hooks/queries';
+import { useTranslation } from 'react-i18next';
+
+export const LatestCommit = (): React.ReactElement => {
+  const { t } = useTranslation();
+  const { org, app } = useStudioEnvironmentParams();
+  const { data: newMasterBranchStatus } = useBranchStatusQuery(org, app, 'master');
```

Comment:
I think a better name for the fetched data is only `masterBranchStatus` since it is not fetched through a refetch anymore